_posts/blog/2018-09-18/2018-09-18-5 -tips-to-make-your-openbazaar-listings-pop.markdown

```diff
@@ -0,0 +1,158 @@
+
+### 2. Take many, many photos 📸
+
+If a picture is worth a thousand words and you can attach up to 30 photos for every listing, that should save you 30,000 words, right? (And way too many headaches.) Take advantage of this.
+
+Photos are arguably the most important part of a listing -- it’s the closest a buyer can come to seeing your product in person. Take clear, high-quality pictures in plenty of light to really let your product shine.
+
+![Simple Listing Image on OpenBazaar](Simple Listing Image on OpenBazaar.jpg "Simple Listing Image on OpenBazaar")
+
+#### Get rid of background clutter
+
+Don’t detract from the star of the show. Stage your product against a plain background. This removes distractions and allows your product to stand out.
+
+> **TIP:** Can’t find a good plain surface anywhere? Textured surfaces can also work well. Place your product on marble counters and wooden coffee tables. Or, just hang up a bedsheet!
+
+#### Give us variety
+
+Show us how your product looks in different angles and settings. Snap some closeups to capture unique and intricate details. Showcase your product in action. If you’re selling a t-shirt, take photos of the shirt on a model so your buyers can see how much cooler it looks on a human being. 
+
+Don’t forget to include photos of variants too! If you’re selling the same dress in red, white, and black, show us what each of them looks like.
+
+![Dress Variations on OpenBazaar](Dress Variations on OpenBazaar.png "Dress Variations on OpenBazaar")
+
+#### Choose your primary photo
+
+Your primary photo is the first thing a buyer will see when they find your listing. It’s both the thumbnail image and the cover photo on your listing’s main page. You want to make a stellar first impression, so use your most captivating photo as your listing’s primary photo. To do this, click your chosen photo and drag it to the front of the gallery.
+
+![Setting Your Primary Listing Photo on OpenBazaar](Setting Your Primary Listing Photo on OpenBazaar.gif "Setting Your Primary Listing Photo on OpenBazaar")
+
+Ideally, your primary photo should capture your product as a whole. So instead of a close-up detail shot, try to choose a photo that shows your product in full view. 
+
+### 3. Give us the deets ☕
+
+Details are key, so don’t skimp on product descriptions! Start with the basics first: what’s for sale? 
+
+- _Handmade red mini tassel earrings_
+- _Vintage whiskey decanter set with matching glasses_
+- _New Age Beets headphones: powered purely by organic beet juice_
+
+Then, go into more detail.
+
+#### Colour
+  
+Colours are deceivingly hard to convey through photos. How a shade looks can vary wildly depending on lighting, or the kind of device you’re using to view it. Give your buyers a more accurate understanding of an item’s colour by using more specific descriptors. 
```

Comment:
Brittish Colours, or American Colors?

Reply:
From Monique so she's bringing that British influence!